elements/rocky-container-pulp/8.6-pulp-mofed

```diff
@@ -0,0 +1,130 @@
+[nfv]
+name=Rocky Linux $releasever - NFV
+enabled=1
+baseurl=https://pulp.128-232-222-245.sslip.io/pulp/content/rocky/8.6/nfv/x86_64/os/20220918T035853/
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-rockyofficial
+Rocky-NFV.repo
+[ceph]
+name=CentOS-$releasever - Ceph Nautilus
+enabled=1
+baseurl=https://pulp.128-232-222-245.sslip.io/pulp/content/centos/8/storage/x86_64/ceph-nautilus/20211122T102435/
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Storage
+ceph.repo
+[docker]
+name=docker-ce-for-centos-8-20211122T102435
+enabled=1
+baseurl=https://pulp.128-232-222-245.sslip.io/pulp/content/docker-ce/centos/8/x86_64/stable/20211122T102435/
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-docker
+docker.repo
+
+RUN dnf update -y --refresh
+
+# Install/remove packages from https://git.rockylinux.org/rocky/kickstarts/-/blob/r8/Rocky-8-GenericCloud.ks
+RUN dnf install -y @core --allowerasing
+RUN dnf install -y chrony dnf yum cloud-init cloud-utils-growpart NetworkManager \
+                dracut-config-generic dracut-norescue firewalld gdisk grub2 \
+                kernel nfs-utils rsync tar dnf-utils yum-utils \
+                python3-jsonschema qemu-guest-agent dhcp-client \
+                rocky-release rng-tools
+RUN dnf erase -y aic94xx-firmware alsa-firmware alsa-lib alsa-tools-firmware \
+              ivtv-firmware iwl100-firmware iwl1000-firmware iwl105-firmware \
+              iwl135-firmware iwl2000-firmware iwl2030-firmware iwl3160-firmware \
+              iwl3945-firmware iwl4965-firmware iwl5000-firmware iwl5150-firmware \
+              iwl6000-firmware iwl6000g2a-firmware iwl6000g2b-firmware iwl6050-firmware \
+              iwl7260-firmware libertas-sd8686-firmware libertas-sd8787-firmware \
+              libertas-usb8388-firmware biosdevname iprutils plymouth
+
+RUN dnf install -y cloud-init libvirt qemu-kvm ceph-common edk2-ovmf qemu-kvm-block-rbd cyrus-sasl docker-ce
+
+# From the base dib element
+RUN dnf install -y findutils util-linux sudo python3 NetworkManager
+
+RUN systemctl unmask console-getty.service dev-hugepages.mount \
+    getty.target sys-fs-fuse-connections.mount systemd-logind.service \
+    systemd-remount-fs.service
+
+RUN curl -L https://www.mellanox.com/downloads/ofed/MLNX_OFED-5.7-1.0.2.0/MLNX_OFED_SRC-5.7-1.0.2.0.tgz --output /tmp/MLNX_OFED_SRC-5.7-1.0.2.0.tgz \
+  && cd /tmp \
+  && tar -xzf MLNX_OFED_SRC-5.7-1.0.2.0.tgz \
+  && dnf install -y perl cmake kernel-devel python36-devel perl-generators python36 python3-Cython rpm-build elfutils-libelf-devel zlib-devel gcc-c++ gdb-headless glib2-devel patch lsof libmnl-devel openssl-devel pciutils-devel pkgconf-pkg-config libstdc++-devel libnl3-devel libtool python3-docutils numactl-devel systemd-devel kernel-rpm-macros glibc-devel pciutils gcc valgrind-devel \
+  && /tmp/MLNX_OFED_SRC-5.7-1.0.2.0/install.pl --hypervisor --kernel 4.18.0-372.26.1.el8_6.x86_64 \
```

Comment:
Is it possible to figure out the installed kernel version - can we assign a variable for this by running a command in the dockerfile?